templates/main.php

```diff
@@ -31,13 +31,11 @@ class="collection"
                     </span>
                     <span class="title">{{ collection.displayname }}</span>
                 </a>
-                <div class="app-navigation-entry-utils">
-                    <ul>
-                        <li class="app-navigation-entry-utils-counter">{{ getCollectionString(collection.id) | counterFormatter }}</li>
-                    </ul>
+                <div class="app-navigation-entry-utils-counter">
```

Comment:
Ah, alright, thanks for the manual link. Would it be allowed to have the counter on the second position to it doesn't get shifted when the three-dots-icon is visible?